packages/core/src/models/demoUsers.ts

```diff
@@ -44,14 +61,19 @@ class DemoUsers {
     return { region, isDemoRegion };
   }
 
+  isDemoStatusSet(user: LabsUser): user is UserWithDemoStatus {
```

Comment:
No matter at all for this but I think this could be `user is LabsUser  & UserWithDemoStatus`. Probably less clear and unnecessary but just thought I'd share as at first I wasn't sure whether TS would infer that LabsUser part ... but it doesn't

Reply:
Great observation. I think this is actually an issue with the UserWithDemoStatus type, which I've now fixed